HISTORY.md

```diff
@@ -10,6 +10,7 @@
 * With FIFO compaction style, options.periodic_compaction_seconds will have the same meaning as options.ttl. Whichever stricter will be used. With the default options.periodic_compaction_seconds value with options.ttl's default of 0, RocksDB will give a default of 30 days.
 * Added an API GetCreationTimeOfOldestFile(uint64_t* creation_time) to get the file_creation_time of the oldest SST file in the DB.
 * An unlikely usage of FilterPolicy is no longer supported. Calling GetFilterBitsBuilder() on the FilterPolicy returned by NewBloomFilterPolicy will now cause an assertion violation in debug builds, because RocksDB has internally migrated to a more elaborate interface that is expected to evolve further. Custom implementations of FilterPolicy should work as before, except those wrapping the return of NewBloomFilterPolicy, which will require a new override of a protected function in FilterPolicy.
+* In cases where TTL is supported, make the default value 30 days. To revert the old behavior, you can explictly set it to 0.
```

Comment:
I think you should mention the default value as UINT64_MAX-1 (or 0xfffffffffffffffe) instead of 30 days here. And RocksDB will internally convert it to 30 days wherever TTL is supported.